js/samples/docs-menu-basic/README.md

```diff
@@ -0,0 +1,27 @@
+## Build it
+
+```
+pnpm build
+```
+
+or if you need to, build everything:
+
+```
+cd </path/to/genkit>; pnpm build; pnpm pack:all; cd -
```

Comment:
Suggest `pnpm run setup` instead of all these separate commands. Also, pack shouldn't be necessary.